package.json

```diff
@@ -15,6 +15,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "yarn rollup:build",
+    "prepare": "npm run rollup:build",
```

Comment:
Why does this not just call build?

Reply:
that was a leftover alias. I added a commit to simplify this.